common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -140,6 +140,14 @@ class CapaFields(object):
         scope=Scope.settings,
         default=default_reset_button
     )
+    require_answer_before_check = Boolean(
+        display_name=_("Require answer before check"),
+        help=_(
+            'Enter true or false. Make it impossible to click "check"/"final check" without selecting a choice.'
```

Comment:
"Enter true or false. If true, learners must enter an answer before they can select "Check" or "Final Check". This setting applies only to checkbox, dropdown, multiple choice, and text, numerical, or math expression input problems."